### DISCUSSION TITLE
Can't figure-out how to use plotters_bitmap::bitmap_pixel::RGBPixel

### DISCUSSION THREAD

**Discussion by TomDeRybel** [plotters-rs/plotters#376] [Q&A] [answered]
Hello,
It's a silly thing but I can't figure out where to find RGBPixel for the bitmap back-end.

Recently, the back-ends were merged into plotters itself. To accommodate for this, I've modded my Cargo.toml as thus:
```
plotters = { git = "https://github.com/plotters-rs/plotters.git", default_features = false, features = ["line_series", "ttf", "bitmap_backend"] }
#plotters-bitmap = { git = "https://github.com/plotters-rs/plotters-bitmap.git" } # No longer needed.
```

And in my source file I have:
```
use plotters::prelude::*;
use plotters_bitmap::bitmap_pixel::RGBPixel;   // <-- This is WRONG.
//use plotters_bitmap::BitMapBackend; // Not needed anymore, is part of the prelude.
```

The issue is that I can't figure-out the path to RBGPixel now...

**Comment by AaronErhardt** [comment, marked as answer]
Looks like only the backend is re-exported and not the crate itself: https://github.com/plotters-rs/plotters/blob/master/plotters/src/lib.rs#L855-L857

Note that the backends weren't merged into plotters but are now simply part of this repo. This means that you still need this (unless plotters-bitmap will be re-exported as whole crate at some point):
```toml
plotters-bitmap = { git = "https://github.com/plotters-rs/plotters.git", default_features = false, features = ["line_series", "ttf", "bitmap_backend"] }
```

**Reply by TomDeRybel** [reply]
Thank you for the fast and clear reply!

I've changed the Cargo.toml based on what you propose and it all works again.

```
plotters = { git = "https://github.com/plotters-rs/plotters.git", default_features = false, features = ["line_series", "ttf", "bitmap_backend"] }
plotters-bitmap = { git = "https://github.com/plotters-rs/plotters.git" }
```
